fynd-platform/GA4-Extension
# ID:FPEX-7939;DONE:100;HOURS:1;Replace Redis Storage to MongoDB Storage

## What type of PR is this? (check all applicable)


## Description

- Remove the dependency on Redis and use MongoDB to store tokens instead.
- Remove extra un-unused config
- Added comments

## Related Tickets & Documents

- Related Issue #16 
- Closes #16 

## QA Instructions, Screenshots, Recordings

Please go ahead and install the extension, and check whether it can work as expected.

## Added/updated tests?

      have not been included_
